test/slice-test.js

```diff
@@ -78,3 +78,14 @@ describe('skipWhile', function () {
     return expectArray([5, 6, 7, 8, 9], s)
   })
 })
+
+describe('skipAfter', function () {
+  it('should skip all elements after the first one for the condition is true', function () {
+    var a = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+    var s = slice.skipAfter(function (x) {
+      return x > 5
```

Comment:
Not critical, but personal pref: I think x === 5 is slightly easier to understand here.  "Skip all items after the first event whose value is greater than 5" seems like a bit more cognitive load for the next person who will read or refactor this test.

Two other potential tests: `skipAfter(alwaysFalse, s)` ~= `s` for 1) `empty()`, and 2) a non-empty stream